app/components/traffic-measure/index.ts

```diff
@@ -139,7 +140,7 @@ export default class TrafficMeasureIndexComponent extends Component<Args> {
       // const relations = await this.trafficMeasureConcept.getOrderedRelations();
     }
 
-    this.signs = new TrackedArray(relatedTrafficSigns);
+    this.signs = A([...relatedTrafficSigns]);
```

Comment:
I would add a comment here that ember-drag-drop expects an ember array, else we might refactor it again in the future :smile: